pkg/types/validation/installconfig_test.go

```diff
@@ -665,7 +665,7 @@ func TestValidateInstallConfig(t *testing.T) {
 				c.Platform.VSphere.VCenter = ""
 				return c
 			}(),
-			expectedError: `^platform\.vsphere.vCenter: Required value: must specify the name of the vCenter$`,
+			expectedError: `^platform\.vsphere.vCenter: Invalid value: "": must specify the name of the vCenter$`,
```

Comment:
i think like the previous error better where we say it is required instead of saying `"" is invalid value`

Reply:
I am using custom validation where "" or "     " or "\n" or "\t" is checked and reported as error. I am not using the defaut valitation, where we just check if the string is of 0 length, at all.